src/CalculatorButtonValues.elm

```diff
@@ -13,13 +13,21 @@ type CalButton
     | CalNumNine
     | CalNumZero
     | CalDecimal
+    | CalAdd
+    | CalEqual
```

Comment:
This are suspiciously similar to `Action`. Maybe you should have a `CalAction Action` constructor instead?